.github/workflows/publish-container-images.yaml

```
@@ -0,0 +1,36 @@
+name: Publish SapMachine container images based on Garden Linux to the SAP GitHub Registry
+
+on:
+  workflow_call:
+    inputs:
+      sapMachineVersion:
+        type: string
+        description: "SapMachine version (e.g. 21)"
+      gardenLinuxVersion:
+        type: string
+        description: "GardenLinux version (e.g. 1592)"        
+jobs:
+  publish_container:
+    name: publish gardenlinux container base image
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      # 1. Checkout the repository
+      - uses: actions/checkout@v4
+      # 2. Run the Python script to fetch and append download stats
```

hmmmmm ... ?!? Copy of Mitja DownloadScript?
```suggestion
      # 2. Publish images
```